buildingsync_asset_extractor/converter.py

```diff
@@ -54,7 +54,7 @@ def convert(value: float, original_type: PowerUnits, to: PowerUnits) -> float:
         'ft-lbf/h': 2655.2237374,
         'ft-lbf/min': 44.253728960000004,
         'Btu/s': 0.00094781,
-        'kBtu/hr': 3.41214163,
+        'kBtu/hr': 341.214163,
```

Comment:
I think it should be 0.00341214163. Same for all below
All the kBtu/hr should multiply 0.001 to compensate the "k" in unit.